app/controllers/admin/volunteer_events_controller.rb

```diff
@@ -2,7 +2,8 @@ class Admin::VolunteerEventsController < Admin::BaseController
 
   def find
     @event = find_event
-    @find_volunteers = Volunteer.left_outer_joins(:events).where( volunteer_events: {:volunteer => nil} )
+    center_volunteers = Volunteer.joins(:preferred_centers).where('preferred_centers.center_id = ?', @event.center_id)
+    @find_volunteers = center_volunteers.left_outer_joins(:events).where( volunteer_events: {:volunteer => nil} )
```

Comment:
it works!
<img width="1033" alt="screen shot 2017-02-15 at 7 57 15 pm" src="https://cloud.githubusercontent.com/assets/21060552/22973545/11075ffc-f3b9-11e6-8c82-71dd5ff6f277.png">